run_querytns.py

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Query TNS for new targets and upload to candidate marshal.
+https://wis-tns.weizmann.ac.il/
+TNS bot apikey must exist in config
+
+.. codeauthor:: Emir Karamehmetoglu <emir.k@phys.au.dk>
+.. codeauthor:: Rasmus Handberg <rasmush@phys.au.dk>
+"""
+
+import argparse
+import logging
+from flows import api, load_config
+import requests
+import json
+from collections import OrderedDict
+from bs4 import BeautifulSoup
+import numpy as np
+from astropy.coordinates import SkyCoord
+import astropy.units as u
+from astropy.time import Time
+from datetime import datetime, timedelta
+import time
+
+
+# Parse command line arguments:
```

Comment:
Wrap all code, which should be run on command-line in `if __name__ == '__main__':` and move all function definition before this.